src/Symfony/Component/Validator/Violation/ConstraintViolationBuilder.php

```diff
@@ -47,7 +47,7 @@ class ConstraintViolationBuilder implements ConstraintViolationBuilderInterface
     /**
      * @param TranslatorInterface $translator
```

Comment:
we should add `@param string $message The error message as a string or a stringable object` here too, to not forget about it

Reply:
And Done.